app/controllers/application_controller.rb

```diff
@@ -6,9 +6,8 @@ class ApplicationController < ActionController::Base
   include GDS::SSO::ControllerMethods
   include Pundit
 
-  before_action :require_signin_permission!
   before_action :set_authenticated_user_header
-  after_action :verify_authorized
```

Comment:
This seems unrelated to GDS::SSO, but instead to pundit. It sounds from the pundit docs that it's useful to have?